lib/agent0/agent0/hyperdrive/exec/get_agent_accounts.py

```diff
@@ -15,12 +14,13 @@
 from web3 import Web3
 from web3.contract.contract import Contract
 
-# pylint: disable=too-many-locals
-
 
+# TODO consolidate config
```

Comment:
What is this TODO for? I was going to make an issue but I need some more details.

Reply:
I wasn't a fan of passing around three different types of configs everywhere, but maybe it's okay, as long as the agent0 interface is concise (which is part of the stateful wrapper for the interface todo). I've updated the comment.